BUILDING.md

```diff
@@ -385,11 +385,9 @@ $ backtrace
 Prerequisites:
 
 * [Python 2.6 or 2.7](https://www.python.org/downloads/)
-* The "Desktop development with C++" workload from
-  [Visual Studio 2017](https://www.visualstudio.com/downloads/) or the
```

Comment:
I asked to remove the link because there is no "Desktop development with C++" link on the current page

Reply:
> I asked to remove the link because there is no "Desktop development with C++" link on the current page

Ok, so it's was just a sub-optimal phrasing...